src/fragments/lib-v1/auth/native_common/getting_started/common.mdx

```diff
@@ -84,4 +84,4 @@ Congratulations! You've successfully setup AWS Cognito Auth plugin.  Check out t
 
 * [Authenticate a user with username password](/lib-v1/auth/signin)
 * [Authenticate a user using a web UI](/lib-v1/auth/signin_web_ui)
-* [Authenticate a user with another auth provider](/lib-v1/auth/social_signin_web_ui)
+* [Authenticate a user with another auth provider](/lib-v1/auth/social)
```

Comment:
Looks like this is the case on lib v2 as well?  It's not broken because there's a redirect from `/social_signin_web_ui` to `/social` but it does act different from the other links on the page, since normally Next.js handles the routing and prevents another page load.

Reply:
After thinking about it, I think it's better to make this change on v2 as well, especially in the case the redirect is accidentally removed

Reply:
done.